src/terminal/scroll/stack.lua

```diff
@@ -0,0 +1,69 @@
+-- Terminal Scroll Stack Module.
```

Comment:
```suggestion
--- Terminal Scroll Stack Module.
```
Doc comment blocks should start with a triple-dash to be picked up by ldoc